services/motion/server_test.go

```diff
@@ -428,6 +428,55 @@ func TestServerMoveOnMapNew(t *testing.T) {
 	})
 }
 
+func TestServerMoveOnMap(t *testing.T) {
```

Comment:
What exactly is this testing? It isn't testing that we plan around obstacles provided in the MoveOnMap request.

Reply:
this tests that if I give nil for obstacles or not nil for obstacles that things still work as expected.
Implementing planning around static obstacles under under the scope of https://viam.atlassian.net/browse/RSDK-6406